en/micro-integrator/docs/references/connectors/amazons3-connector/1.x/amazons3-connector-1.x-example.md

```diff
@@ -26,13 +26,13 @@ If you do not want to configure this yourself, you can simply [get the project](
 
 ## Setting up the environment 
 
-Please follow the steps mentioned at [Setting up Amazon S3](amazons3-connector-config.md) document in order to create a Amazon S3 account and obtain credentials you need to access the Amazon APIs. Keep them saved to be used in the next steps.  
+Please follow the steps mentioned at [Setting up Amazon S3](../amazons3-connector-1.x-config) document in order to create a Amazon S3 account and obtain credentials you need to access the Amazon APIs. Keep them saved to be used in the next steps.  
```

Comment:
I assumed this will be at the same level. Why the extra "../"?

Reply:
yup